core/src/main/java/com/crawljax/core/state/InMemoryStateFlowGraph.java

```diff
@@ -5,7 +5,6 @@
 
 import java.io.Serializable;
 import java.util.*;
```

Comment:
Could this wildcard import go or be replaced appropriately? (Since you're doing some cleanup in this class anyway)